sycl/source/detail/program_manager/program_manager.hpp

```diff
@@ -399,6 +401,8 @@ class ProgramManager {
 
   std::set<std::string> m_KernelUsesAssert;
 
+  bool m_KernelUsesAsan;
```

Comment:
This is confusing to me. `ProgramManager` isn't associated with just a single kernel. See for example `m_KernelUsesAssert` which is a set of names of kernels that have been found to use asserts, yet this is just a single boolean. Should this be a set too, to allow us to check specific kernels? If not, could we make the intention clearer?

Either way, please add a comment specifying what the member indicates, similar to (most) other members.

Reply:
Thanks for you suggestion, done